torch/onnx/utils.py

```diff
@@ -222,8 +222,14 @@ def _scalar(x):
 
 
 def _newNode(g, opname, outputs, *args, **kwargs):
-    aten = kwargs.pop("aten", False)
-    n = g.create(opname, args, outputs)
+    if "::" in opname:
+        aten = False
+        ns_opname = opname
+    else:
+        aten = kwargs.pop("aten", False)
+        ns = "aten" if aten else "onnx"
+        ns_opname = ns + "::" + opname
```

Comment:
Hmm I think it would be cleaner if we had only one way to select the namespace. I think kwargs are slightly more convenient (with defaults pointing to the most common use case).

Reply:
I guess I can refactor the aten code some more

Reply:
I'm going to save this for later.